test/test_jit.py

```diff
@@ -6815,6 +6809,9 @@ def post_add_test(test_name, skipTestIf, do_test):
     for skip in skipTestIf:
         do_test = skip(do_test)
 
+    if test_name in EXCLUDE_SCRIPT or test_name in EXCLUDE_TRACED:
```

Comment:
This change deletes the distinction between EXCLUDE_SCRIPT and EXCLUDE_TRACED, and is disabling tests that were only excluded in one or the other.

Reply:
I noticed that, would it be worthwhile to break up the script testing and tracer testing into separate unit tests? If not then you can't tell which is failing (some of the excluded tests pass as script functions but not as traced functions)